app/assets/javascripts/admin_legacy.js

```diff
@@ -0,0 +1,14 @@
+//= require govuk_publishing_components/lib
+//= require jquery-ui
+//= require extensions_legacy/jquery.distinct
+//= require vendor_legacy/jquery/chosen.jquery.js
+//= require vendor_legacy/jquery/jquery.player.min.js
+//= require vendor_legacy/jquery/jquery.highlighttextarea.js
+//= require vendor_legacy/jquery/jquery-debounce.js
+//= require vendor_legacy/google-diff-match-patch.js
+//= require ./application_legacy/convert-html-pub-charts
+//= require govuk_legacy
+//= require_tree ./common_legacy
+//= require ./application_legacy/hide-extra-rows.js
```

Comment:
application_legacy seems a bit of a strange folder name. Do we know what the purpose of the application folder is? it seems like they should either be in common, if they're shared with frontend and backend, or in admin_legacy if they're just needed for admin